src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -1859,91 +1879,7 @@ Connection connect(Properties propsIn, SQLServerPooledConnection pooledConnectio
             loginTimeoutSeconds *= 10;
```

Comment:
`loginTimeoutSeconds` doesn't look like it's used. We should probably remove it.